viewer/packages/camera-manager/src/StationaryCameraManager.ts

```diff
@@ -285,7 +285,7 @@ export class StationaryCameraManager implements CameraManager {
   private getCursorRay(event: WheelEvent) {
     const { width, height } = this._domElement.getBoundingClientRect();
     const { offsetX, offsetY } = clickOrTouchEventOffset(event, this._domElement);
-    const ndcCoordinates = pixelToNormalizedDeviceCoordinates(offsetX, offsetY, width, height);
+    const ndcCoordinates = getNormalizedPixelCoordinatesBySize(offsetX, offsetY, width, height);
```

Comment:
If `getNormalizedPixelCoordinatesBySize` is not used anywhere else within `Reveal`, we can remove it from `wotdToViewport.ts` file and use `getNormalizedPixelCoordinates` here as `this._domElement` is available.

Reply:
It is also used by pixelToNormalizedDeviceCoordinates. So I want to keep it as it is now.